### PR TITLE
[DO NOT REVIEW] Merge `RBA Metadata` feature branch

### PR DESCRIPTION
**Pull request by richherrera**
### Summary of changes

- These changes were already reviewed as sub-PRs
- This PR is to trigger CI checks 

### Checklist

 - [x] Added a changelog entry
 - [x] Relevant test coverage

### Authors

- @sarahkoop @scannillo @tdchow @jaxdesmarais

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Braintree Android SDK Release Notes
 
+## unreleased
+
+* Breaking Changes
```
Comment (jaxdesmarais) [inline CHANGELOG.md:5]: I don't think this should be a breaking change

File: CHANGELOG.md [hunk 2]
```diff
@@ -30,7 +36,6 @@
   * PayPal
     * Make `PayPalPaymentAuthRequestParams` internal
     * Remove `PayPalPaymentAuthResultInfo`
-    * Rename `PayPalLauncher.handleReturnToAppFromBrowser()` to `PayPalLauncher.handleReturnToApp()`
```
Comment (jaxdesmarais) [inline CHANGELOG.md:33]: Should this be removed?

Comment (richherrera) [reply CHANGELOG.md]: Good catch, it was removed at some point. Reverted it, thanks! [0f21f4d](https://github.com/braintree/braintree_android/pull/1174/commits/0f21f4df557fb4556a4b6a756e76508bd3bbb071)

File: CHANGELOG.md (hunk 1 above)
Comment (richherrera) [reply CHANGELOG.md]: Thanks for the heads up. I just moved the entire block, but I should have confirmed first. I’ll keep that in mind for the future. Updated: [0f21f4d](https://github.com/braintree/braintree_android/pull/1174/commits/0f21f4df557fb4556a4b6a756e76508bd3bbb071)

**Review by jaxdesmarais — APPROVED** [review]

**Review by scannillo — APPROVED** [review]

**Pull request merged**